src/util.tsx

```diff
@@ -141,13 +141,15 @@ export function calcDropPosition(
     }
   }
 
+  const abstractDragDataNode = keyEntities[dragNode.props.eventKey].node;
```

Comment:
`dargNode` is passed from arguments, why get it from the map?

Reply:
The argument one is `NodeInstance`. and i think the useful one to user is `DataNode`

Reply:
can't we get datanode from nodeinstance?

Reply:
Yes, I think get it from arg but not map is clearer.